server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +624,42 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Set up listener to forward signal to clients in the collaboration session when the broadcast-signal endpoint is called
+			collborationSessionEventEmitter?.on(
+				"broadcastSignal",
+				(broadcastSignal: IBroadcastSignalEventPayload) => {
+					const { tenantId, documentId } = broadcastSignal;
+					const roomToBroadcastSignal: IRoom = { tenantId, documentId };
+
+					// No-op if the room (collab session) that signal came in from is different
+					// than the current room. We reuse websockets so there could be multiple rooms
+					// that we are sending the signal to, and we don't want to do that.
+					if (
+						roomToBroadcastSignal.documentId === room.documentId &&
+						roomToBroadcastSignal.tenantId === room.tenantId
+					) {
+						const contents = JSON.parse(
```

Comment:
Let's add some protection around this JSON.parse. If `signalContent` is malformed, this will throw an `Uncaught Exception` which will crash Alfred. I'm guessing signalContent is from the client, so that would open us up to an easy attack vector.
<img width="573" alt="image" src="https://github.com/microsoft/FluidFramework/assets/14362200/14211f92-b2ab-465e-9d6b-c50e63eb0e18">

Reply:
I think a simple `try {} catch { log & return}` would suffice